flutter/lib/src/default_integrations.dart

```diff
@@ -195,15 +196,95 @@ class _LoadContextsIntegrationEventProcessor extends EventProcessor {
         event = event.copyWith(contexts: eventContexts);
       }
 
-      if (infos['integrations'] != null) {
-        final integrations = List<String>.from(infos['integrations'] as List);
+      final tagsMap = infos['tags'];
+      if (tagsMap != null) {
+        final tags = event.tags ?? {};
+        final newTags = Map<String, String>.from(tagsMap as Map);
+
+        for (final tag in newTags.entries) {
+          if (!tags.containsKey(tag.key)) {
+            tags[tag.key] = tag.value;
+          }
+        }
+        event = event.copyWith(tags: tags);
+      }
+
+      final extraMap = infos['extra'];
+      if (extraMap != null) {
+        final extras = event.extra ?? {};
+        final newExtras = Map<String, dynamic>.from(extraMap as Map);
+
+        for (final extra in newExtras.entries) {
+          if (!extras.containsKey(extra.key)) {
+            extras[extra.key] = extra.value;
+          }
+        }
+        event = event.copyWith(extra: extras);
+      }
+
+      final userMap = infos['user'];
+      if (event.user == null && userMap != null) {
+        final user = Map<String, dynamic>.from(userMap as Map);
+        event = event.copyWith(user: SentryUser.fromJson(user));
+      }
+
+      final distString = infos['dist'];
+      if (event.dist == null && distString != null) {
+        event = event.copyWith(dist: distString as String);
+      }
+
+      final environmentString = infos['environment'];
+      if (event.environment == null && environmentString != null) {
+        event = event.copyWith(environment: environmentString as String);
+      }
+
+      final fingerprintList = infos['fingerprint'];
+      if (fingerprintList != null) {
+        final eventFingerprints = event.fingerprint ?? [];
+        final newFingerprint = List<String>.from(fingerprintList as List);
+
+        for (final fingerprint in newFingerprint) {
+          if (!eventFingerprints.contains(fingerprint)) {
+            eventFingerprints.add(fingerprint);
+          }
+        }
+        event = event.copyWith(fingerprint: eventFingerprints);
+      }
+
+      final levelString = infos['level'];
+      if (event.level == null && levelString != null) {
+        event = event.copyWith(level: SentryLevel.fromName(levelString));
+      }
+
+      final breadcrumbsList = infos['breadcrumbs'];
```

Comment:
We need to be sure that we don't duplicate the crumbs, and the timeline of the crumbs should make sense.

Reply:
Right now the breadcrumbs form the native bridge are merged with the hybrid ones and sorted by date, is there anything else we need to check?